diff --git a/engines/configure.engines b/engines/configure.engines
index a52276130a..53df74f29c 100644
--- a/engines/configure.engines
+++ b/engines/configure.engines
@@ -10,6 +10,7 @@ add_engine cge "CGE" yes
 add_engine cine "Cinematique evo 1" yes
 add_engine composer "Magic Composer" yes
 add_engine cruise "Cinematique evo 2" yes
+add_engine cryo "Cryo" no
 add_engine draci "Dragon History" yes
 add_engine drascula "Drascula: The Vampire Strikes Back" yes
 add_engine dreamweb "Dreamweb" yes
diff --git a/engines/engines.mk b/engines/engines.mk
index bcf97df991..7556641f7e 100644
--- a/engines/engines.mk
+++ b/engines/engines.mk
@@ -46,6 +46,11 @@ DEFINES += -DENABLE_CRUISE=$(ENABLE_CRUISE)
 MODULES += engines/cruise
 endif
 
+ifdef ENABLE_CRYO
+DEFINES += -DENABLE_CRYO=$(ENABLE_CRYO)
+MODULES += engines/cryo
+endif
+
 ifdef ENABLE_DRACI
 DEFINES += -DENABLE_DRACI=$(ENABLE_DRACI)
 MODULES += engines/draci
diff --git a/engines/plugins_table.h b/engines/plugins_table.h
index e5ac5ef..2b4bfcc 100644
--- a/engines/plugins_table.h
+++ b/engines/plugins_table.h
@@ -20,6 +20,9 @@ LINK_PLUGIN(COMPOSER)
 #if PLUGIN_ENABLED_STATIC(CRUISE)
 LINK_PLUGIN(CRUISE)
 #endif
+#if PLUGIN_ENABLED_STATIC(CRYO)
+LINK_PLUGIN(CRYO)
+#endif
 #if PLUGIN_ENABLED_STATIC(DRACI)
 LINK_PLUGIN(DRACI)
 #endif
